README.md

```diff
@@ -25,6 +25,9 @@
 - LinkedIn: [LinkedIn](https://linkedin.com/in/victoir-barh)
 
 ## Getting Started
+- git clone to clone the project 
+- npm install to install all dependecies
+- npm start to run the server
```

Comment:
- [ ] This project uses webpack to compile the code, it will be nice to add necessary instructions on how to run the app, commands like npm run build are necessary for someone to get started with the project.